modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/ObservationProgressDisplay.scala

```diff
@@ -89,15 +90,22 @@ object SmoothProgressBar {
         else if (remaining > 0) s"${p.fileId} - $durationStr left"
         else s"${p.fileId} - Reading out..."
 
-      Progress(Progress.Props(
-        label       = label,
-        total       = p.total,
-        value       = s.value,
-        color       = "blue".some,
-        progressCls = List(SeqexecStyles.observationProgressBar),
-        barCls      = List(SeqexecStyles.observationBar),
-        labelCls    = List(SeqexecStyles.observationLabel)
-      ))
+      if(p.hideBar)
```

Comment:
I guess this is to hide the bar for N&S right?

Reply:
Exactly. The logic to build the message to display is in (previously named) `ObservationProgressBar` and `SmoothProgressBar`, which displayed the message as the progress bar label.

Extracting all the message building logic seemed quite complex, so I opted to rename the classes `*ProgressBar` = `*ProgressDisplay` and make showing the progress bar optional.